charts/merlin/templates/_helpers.tpl

```diff
@@ -76,17 +76,32 @@ Generated names
     {{- end -}}
 {{- end -}}
 
+{{/*
+Application version
+*/}}
+{{- define "merlin.version" -}}
```

Comment:
Can you add test cases [here](https://github.com/caraml-dev/helm-charts/blob/main/charts/merlin/tests/merlin_deployment_test.yaml#L9) to cover this block for the deployment resource?